libraries/shared/chunker/log_chunker.go

```diff
@@ -44,27 +42,25 @@ func NewLogChunker() *LogChunker {
 	}
 }
 
-// Configures the chunker by adding more addreses and topics to consider.
-func (chunker *LogChunker) AddConfigs(transformerConfigs []shared_t.EventTransformerConfig) {
-	for _, config := range transformerConfigs {
-		for _, address := range config.ContractAddresses {
-			var lowerCaseAddress = strings.ToLower(address)
-			chunker.AddressToNames[lowerCaseAddress] = append(chunker.AddressToNames[lowerCaseAddress], config.TransformerName)
-			chunker.NameToTopic0[config.TransformerName] = common.HexToHash(config.Topic)
-		}
+// Configures the chunker by adding one config with more addresses and topics to consider.
+func (chunker *LogChunker) AddConfig(transformerConfig transformer.EventTransformerConfig) {
+	for _, address := range transformerConfig.ContractAddresses {
+		var lowerCaseAddress = strings.ToLower(address)
+		chunker.AddressToNames[lowerCaseAddress] = append(chunker.AddressToNames[lowerCaseAddress], transformerConfig.TransformerName)
+		chunker.NameToTopic0[transformerConfig.TransformerName] = common.HexToHash(transformerConfig.Topic)
 	}
 }
 
 // Goes through an array of logs, associating relevant logs (matching addresses and topic) with transformers
-func (chunker *LogChunker) ChunkLogs(logs []types.Log) map[string][]types.Log {
-	chunks := map[string][]types.Log{}
+func (chunker *LogChunker) ChunkLogs(logs []core.HeaderSyncLog) map[string][]core.HeaderSyncLog {
+	chunks := map[string][]core.HeaderSyncLog{}
 	for _, log := range logs {
 		// Topic0 is not unique to each transformer, also need to consider the contract address
-		relevantTransformers := chunker.AddressToNames[strings.ToLower(log.Address.String())]
+		relevantTransformers := chunker.AddressToNames[strings.ToLower(log.Log.Address.Hex())]
```

Comment:
We're grabbing untransformed logs out of the db in the delegate step, and those rows have the associated `header_id` and `log_id` for the event.

If it weren't in one struct, I _think_ it would require some overhead in the code to pass around a `types.Log` along with its `header_id` and `log_id` while keeping them separate. Eh?

Reply:
@rmulhol Makes sense! I have the same feeling, but I think I recall seeing more places where we pass `core.Log` + `headerID` where this might have simplified :thinking:

Reply:
Will take another look. Definitely shouldn't need to pass a `core.HeaderSyncLog` + `headerID` separately anywhere